lambroll.go

```diff
@@ -267,7 +267,7 @@ func newFunctionFrom(c *lambda.FunctionConfiguration, code *lambda.FunctionCodeL
 			Variables: e.Variables,
 		}
 	}
-	if i := c.ImageConfigResponse; i != nil {
+	if i := c.ImageConfigResponse; i != nil && *i != (lambda.ImageConfigResponse{}) {
```

Comment:
[ImageConfigResponse](https://pkg.go.dev/github.com/aws/aws-sdk-go/service/lambda#ImageConfigResponse) has Error and ImageConfig struct fields. So even if ImageConfigResponse is not a zero value, ImageConfigResponse.ImageConfig may be nil.

I suggest checking it is not nil as follows.

```suggestion
	if i := c.ImageConfigResponse; i != nil && i.ImageConfig != nil {
```

Reply:
Thank you for your suggestion! I checked to run the `init` sub command successfully when after applying this change on my laptop.

```console
$ go run ./cmd/lambroll/main.go init --function-name=testtest
2023/10/19 15:09:26 [info] lambroll current with function.json
2023/10/19 15:09:26 [info] function testtest found
2023/10/19 15:09:27 [info] creating .lambdaignore
2023/10/19 15:09:27 [info] creating function.json
2023/10/19 15:09:27 [info] completed
```